test/storage.js

```diff
@@ -0,0 +1,55 @@
+var _Promise = require('bluebird');
+
+var chai = require('chai');
+var sinon = require('sinon');
+
+var errors = require('../api/v1/errors');
+var utils = require('../api/v1/utils');
+var StorageService = require('../api/v1/services/StorageService.js');
+var Upload = require('../api/v1/models/Upload.js');
+var User = require('../api/v1/models/User.js');
+var client = require('aws-sdk');
+
+var assert = chai.assert;
+var expect = chai.expect;
+
+describe('StorageService',function(){
+
+    describe('createUpload', function () {
+        var _forge;
```

Comment:
Let's allow this for now. This is a sort of system-dependent variable -- if it's modified directly, the tests will fail (until restore is called).